user_data/events.csv

```diff
@@ -0,0 +1,53 @@
+subject,session_datetime,trial_id,event_start_time,event_type
+subject3,2021-10-25 13:06:40,1,2.794,center
+subject3,2021-10-25 13:06:40,1,13.252,center
+subject3,2021-10-25 13:06:40,1,16.014,left
+subject3,2021-10-25 13:06:40,3,52.128,left
+subject3,2021-10-25 13:06:40,3,59.234,center
+subject3,2021-10-25 13:06:40,3,65.688,left
+subject3,2021-10-25 13:06:40,4,76.33,center
+subject3,2021-10-25 13:06:40,4,82.185,center
+subject3,2021-10-25 13:06:40,4,89.928,center
+subject3,2021-10-25 13:06:40,5,95.942,right
+subject3,2021-10-25 13:06:40,5,102.981,center
+subject3,2021-10-25 13:06:40,9,195.879,right
+subject3,2021-10-25 13:06:40,10,216.035,left
+subject3,2021-10-25 13:06:40,11,246.891,left
+subject3,2021-10-25 13:06:40,14,317.824,center
+subject3,2021-10-25 13:06:40,14,322.133,center
+subject3,2021-10-25 13:06:40,15,340.808,center
+subject3,2021-10-25 13:06:40,16,366.043,left
+subject3,2021-10-25 13:06:40,16,369.579,center
+subject3,2021-10-25 13:06:40,16,375.78,right
+subject3,2021-10-25 13:06:40,19,435.642,center
+subject3,2021-10-25 13:06:40,19,442.807,right
+subject3,2021-10-25 13:06:40,22,510.049,center
+subject3,2021-10-25 13:06:40,22,516.709,left
+subject3,2021-10-25 13:06:40,22,524.809,center
+subject3,2021-10-25 13:06:40,23,532.33,right
+subject3,2021-10-25 13:06:40,23,537.235,center
+subject3,2021-10-25 13:06:40,24,557.895,center
+subject3,2021-10-25 13:06:40,24,565.882,left
+subject3,2021-10-25 13:06:40,26,604.378,left
+subject3,2021-10-25 13:06:40,26,612.176,center
+subject3,2021-10-25 13:06:40,26,620.408,right
+subject3,2021-10-25 13:06:40,29,674.704,center
+subject3,2021-10-25 13:06:40,31,728.045,center
+subject3,2021-10-25 13:06:40,33,776.118,right
+subject3,2021-10-25 13:06:40,33,783.004,center
+subject3,2021-10-25 13:06:40,33,788.685,right
+subject3,2021-10-25 13:06:40,34,796.592,left
+subject3,2021-10-25 13:06:40,34,804.171,right
+subject3,2021-10-25 13:06:40,35,823.11,center
+subject4,2021-10-25 13:06:40,35,827.758,right
+subject5,2021-10-25 13:06:40,35,834.199,right
+subject6,2021-10-25 13:06:40,36,847.586,center
+subject7,2021-10-25 13:06:40,36,850.907,right
+subject8,2021-10-25 13:06:40,37,871.637,center
+subject9,2021-10-25 13:06:40,37,872.803,left
+subject10,2021-10-25 13:06:40,37,882.717,center
+subject11,2021-10-25 13:06:40,39,914.219,right
+subject12,2021-10-25 13:06:40,39,920.791,right
+subject13,2021-10-25 13:06:40,40,937.419,right
+subject14,2021-10-25 13:06:40,40,946.424,center
+subject15,2021-10-25 13:06:40,40,953.378,left
```

Comment:
Hi @CBroz1, was adding other subjects here intentional?  If so, please ignore my comment.

Reply:
I did not! good catch! - click/drag error with excel being 'smarter' than it should